cypress/support/contacts/tabs/overviewTab .js

```diff
@@ -1,11 +1,14 @@
-import EditIndividualModal from "../modals/editIndividual"
+import { ConactPageEditIndividualModal } from "../modals/editIndividual"
 
-export default class ContactPageOverviewTab {
-
-    EditIndividual = new EditIndividualModal()
+export class ContactPageOverviewTab {
+    constructor() {
+        this.EditIndividual = new ConactPageEditIndividualModal()
+    }
```

Comment:
Another way to make a constructor. Just made for practice and education purposes.